vm/devices/vmbus/vmbus_client/src/lib.rs

```diff
@@ -55,7 +55,11 @@ use zerocopy::KnownLayout;
 const SINT: u8 = 2;
 const VTL: u8 = 0;
 const SUPPORTED_VERSIONS: &[Version] = &[Version::Iron, Version::Copper];
-const SUPPORTED_FEATURE_FLAGS: FeatureFlags = FeatureFlags::all();
+const SUPPORTED_FEATURE_FLAGS: FeatureFlags = FeatureFlags::new()
```

Comment:
Can we go further here and remove the `all` and `contains_unsupported_bits` methods and move their definitions to the individual crates as necessary? Right now these are in `vmbus_core`, which leads to problems like this one.